ingestion/tests/unit/topology/database/test_glue.py

```diff
@@ -0,0 +1,381 @@
+                "skip.header.line.count": "2",
+                "transient_lastDdlTime": "1652446944",
+            },
+            "CreatedBy": "arn:aws:iam::118146679784:user/abhishek-pandey",
+            "IsRegisteredWithLakeFormation": False,
+            "CatalogId": "118146679784",
+        },
+        {
+            "Name": "map_table",
+            "DatabaseName": "default",
+            "Owner": "hadoop",
+            "CreateTime": datetime.datetime(2022, 5, 27, 14, 13, 20),
+            "UpdateTime": datetime.datetime(2022, 5, 27, 14, 13, 20),
+            "LastAccessTime": datetime.datetime(1970, 1, 1, 5, 30),
+            "Retention": 0,
+            "StorageDescriptor": {
+                "Columns": [{"Name": "c1", "Type": "map<string,int>"}],
+                "Location": "s3://athena-postgres/map-test",
+                "InputFormat": "org.apache.hadoop.mapred.TextInputFormat",
+                "OutputFormat": "org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat",
+                "Compressed": False,
+                "NumberOfBuckets": -1,
+                "SerdeInfo": {
+                    "SerializationLibrary": "org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe",
+                    "Parameters": {"serialization.format": "1"},
+                },
+                "BucketColumns": [],
+                "SortColumns": [],
+                "Parameters": {},
+                "SkewedInfo": {
+                    "SkewedColumnNames": [],
+                    "SkewedColumnValues": [],
+                    "SkewedColumnValueLocationMaps": {},
+                },
+                "StoredAsSubDirectories": False,
+            },
+            "PartitionKeys": [],
+            "TableType": "EXTERNAL_TABLE",
+            "Parameters": {"EXTERNAL": "TRUE", "transient_lastDdlTime": "1653641000"},
+            "CreatedBy": "arn:aws:iam::118146679784:user/pere-brull",
+            "IsRegisteredWithLakeFormation": False,
+            "CatalogId": "118146679784",
+        },
+    ],
+    "ResponseMetadata": {
+        "RequestId": "c0524fd8-cba1-44c3-915d-fbeccc9b3334",
+        "HTTPStatusCode": 200,
+        "HTTPHeaders": {
+            "date": "Tue, 26 Jul 2022 17:55:29 GMT",
+            "content-type": "application/x-amz-json-1.1",
+            "content-length": "5247",
+            "connection": "keep-alive",
+            "x-amzn-requestid": "c0524fd8-cba1-44c3-915d-fbeccc9b3334",
+        },
+        "RetryAttempts": 0,
+    },
+}
+
+
+EXPECTED_DATABASE_NAMES = ["118146679784"]
+
+EXPECTED_DATABASE_SCHEMA_NAMES = [
+    "default",
+    "mydatabase",
+    "testdatalake_db",
+    "zipcode-db",
+]
+
+EXPECTED_TABLE_NAMES = ["cloudfront_logs", "cloudfront_logs2", "map_table"]
+
+EXPECTED_TABLE_TYPES = [TableType.External, TableType.Iceberg, TableType.View]
+
+
+class GlueUnitTest(TestCase):
+    def __init__(self, methodName: str = ...) -> None:
+        super().__init__(methodName)
+        self.init_workflow()
```

Comment:
@OnkarVO7 can we mock the methods in init instead of calling `init_workflow` here,  please refer to [test_fivetran.py](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/tests/unit/topology/pipeline/test_fivetran.py)

Reply:
okay I'll take a look